.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  ci:
+    name: >
+      Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    container:
+      image: erlang:${{matrix.otp_vsn}}
+    strategy:
+      matrix:
+        otp_vsn: [19.0, 19.3,
+                  20.0, 20.1, 20.3,
+                  21.0, 21.1, 21.2, 21.3,
+                  22.0, 22.1, 22.2,
+                  22.3]
```

Comment:
^ Since we're moving to GHA, we may as well and once more start running continuous integration on OTP versions 19.1, 19.2 and 20.2.

The only reason why they were disabled on Travis is because they stopped being provided at some point.

Reply:
93c3273